src/docs/markdown/api.md

```diff
@@ -246,7 +246,9 @@ which is much easier to remember and write by hand.
 ## Concurrent config changes
 
 <aside class="tip">
-	This section is for all <code>/config/</code> endpoints. It is experimental and subject to change.
+
+	This section is for all `/config/` endpoints. It is experimental and subject to change.
```

Comment:
I think you need to remove the leading tab too otherwise it gets parsed as a code block; needs to be gutter aligned as markdown

Reply:
Ah... that's annoying. I might just keep it as HTML. It's not that ugly in this case. Thanks

Reply:
Hard disagree, it's sooo much better because we can do more complex stuff in there. Just put extra newlines before/after the `<aside>` and it reads ok IMO